mne_bids_pipeline/_config.py

```diff
@@ -1,4 +1,4 @@
-# Default settings for data processing and analysis.
+"""Default settings for data processing and analysis."""
```

Comment:
... and I'm going to revert this since it injects this line at the top of each rendered `.md` file